WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -254,6 +255,7 @@ public WordPressDB(Context ctx) {
         SiteSettingsTable.createTable(db);
         CommentTable.createTables(db);
         SuggestionTable.createTables(db);
+        NotificationsTable.createTables(db);
 
         // Update tables for new installs and app updates
         int currentVersion = db.getVersion();
```

Comment:
Shouldn't we also be migrating the current DB schema and `delete` Simperium DB? something like what happens in version 36:

````
            case 36:
                // Delete simperium DB again - from 4.6.1 to 4.7
                // Fix a sync issue happening for users who have both wpios and wpandroid active clients
                ctx.deleteDatabase("simperium-store");
                currentVersion++;

```